cionic/api.py

```diff
@@ -374,21 +377,86 @@ def download_file(destpath: str, url: str, headers: dict = None) -> bool:
     return True
 
 
+def download_npz_from_metadata(
+    orgid: str,
+    studyid: str,
```

Comment:
these args say `id` but the usage looks more like it's using a shortname. sorry for the semantics but I think a backend eng would see `org/studyid` and expect a number instead of a string - I recommend calling it `org/studyshortname` for clarity (or at least update the comment for `orgid`)